data_adapter_oemof/adapters.py

```diff
@@ -34,7 +36,10 @@ def get_default_parameters(self, struct: dict, mapper: Mapper) -> dict:
             defaults.update(
                 {
                     "name": calculations.get_name(
-                        mapper.get("region"), mapper.get("carrier"), mapper.get("tech")
+                        mapper.get("region"),
+                        mapper.get("carrier"),
+                        mapper.get("tech"),
+                        self.counter,
```

Comment:
Why dont we pass the process name? `defaults.update({"name": mapper.process_name})`

Reply:
Good idea. Counting must still be added

Reply:
I added it to #71